src/lecturer/services/backEnd/BackEndService.tsx

```diff
@@ -29,10 +29,7 @@ const createLecture = () => {
     return fetch(`${BASE_URL}/lectures`, {
         method: "POST",
         //TODO fix cors
-        mode: 'cors',
-        headers: {
```

Comment:
czy headers było od początku niepotrzebne, czy teraz się takie stało?

Reply:
They were unnecessary.
I thought that CORS is a client-side problem and headers should be set here.